<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationScreen</class>
 <widget class="QWidget" name="RegistrationScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
}
QPushButton:focus
{ 	background-color: white; }

QPushButton:hover:!pressed
{
 	background-color: rgba(203, 223, 242, 70);
}

QPushButton:pressed 
{
     background-color: rgba(156, 215, 249, 190);     
}

QWidget#DataPreparationlScreen
{
     background-color: rgb(255, 255, 255);     
}


QPushButton{
	background-color: rgb(225, 225, 225);
}
QPushButton:focus{ 	
	background-color: white; 
}

QPushButton:hover:!pressed
{
 	background-color: rgba(203, 223, 242, 70);
}

QPushButton:pressed 
{
     background-color: rgba(156, 215, 249, 190);     
}

QWidget#ManualScreen
{
     background-color: rgb(255, 255, 255);     
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>19</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(47, 82, 143);</string>
   </property>
   <property name="text">
    <string>Passion Fruit Classification [Online]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-iconSize: 70px; font-size:20px</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/exit_green.png</normaloff>../resources/icons/exit_green.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>106</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/icons/passion_fruit.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>1238</width>
     <height>6</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(2, 34, 71);
background-color: rgb(32, 74, 135);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
border: 1px solid gray;
border-color: gray;
font-size: 16px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
}
QGroupBox::title {
padding: 5px 133px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
background-color: rgb(47, 82, 143);
color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="title">
    <string>Input and control</string>
   </property>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 10px;
border: 1px solid rgb(150, 150, 150);
font: 18pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>861</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
border: 1px solid gray;
border-color: gray;
font-size: 16px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
}
QGroupBox::title {
padding: 5px 401px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
background-color: rgb(47, 82, 143);
color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="title">
    <string>Classification Result</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Lane 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lane 1:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="classifyLane1Box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    selection-background-color: rgb(114, 159, 207);
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VIP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="classifyLane2Box">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    selection-background-color:  rgb(114, 159, 207);
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VIP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Processing Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="progStatus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>WAITING</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>1241</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
border: 1px solid gray;
border-color: gray;
font-size: 16px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
}
QGroupBox::title {
padding: 5px 527px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
background-color: rgb(47, 82, 143);
color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="title">
    <string>    Camera View    </string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="cam1Pixmap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/images/frame.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cam2Pixmap">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/images/thumbnail.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cam3Pixmap">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/images/thumbnail.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
background-color: rgb(50,50, 50);</string>
    </property>
    <property name="text">
     <string>Lane 1 - Detection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
background-color: rgb(50,50, 50);</string>
    </property>
    <property name="text">
     <string>Lane 2 - Detection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="camViewLane1" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="baseCameraView" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="camViewLane2" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
background-color: rgb(50,50, 50);</string>
    </property>
    <property name="text">
     <string>Live View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>cam1Pixmap</zorder>
   <zorder>cam2Pixmap</zorder>
   <zorder>cam3Pixmap</zorder>
   <zorder>camViewLane1</zorder>
   <zorder>baseCameraView</zorder>
   <zorder>camViewLane2</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_5</zorder>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>label</zorder>
  <zorder>exitButton</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
